mcs/class/System.Drawing/System.Drawing/Graphics.cs

```diff
@@ -2443,12 +2443,27 @@ public void TranslateTransform (float dx, float dy, MatrixOrder order)
 			}
 		}
 
-		[MonoTODO]
 		[EditorBrowsable (EditorBrowsableState.Never)]
 		public object GetContextInfo ()
 		{
-			// only known source of information @ http://blogs.wdevs.com/jdunlap/Default.aspx
-			throw new NotImplementedException ();
+			Region clip = this.Clip;
+			Matrix transform = this.Transform;
+			PointF pointF = PointF.Empty;
+			PointF empty = PointF.Empty;
+
+			if (!transform.IsIdentity) {
+				float[] elements = transform.Elements;
+				pointF.X = elements[4];
+				pointF.Y = elements[5];
+			}
```

Comment:
This is not used anywhere.

I think the whole point of any post-transformation of `Clip` and `Transform` is to handle interaction with `BeginContainer`, which is not done anyway. So, this method should either do just `return new object[] { Clip, Transform };` and have a FIXME comment about interaction with `BeginContainer` not being implemented, or it should have proper unit tests to verify that it really does what it is supposed to do.